website/blog/2018-07-19-whats-happening-with-the-pipeline-proposal.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title:  "What's Happening With the Pipeline (|>) Proposal?"
+author: James DiGioia
+authorURL: https://twitter.com/JamesDiGioia
+date: 2018-07-19 12:00:00
+categories: announcements
+share_text: "What's Happening With the Pipeline (|>) Proposal?"
+---
+
+With the release of babel@7.0.0-beta54, we [introduced](https://github.com/babel/babel/pull/3159) a new required configuration flag to `@babel/plugin-proposal-pipeline-operator`, a breaking change not for the pipeline operator. To clear up the confusion, let's take a look at the pipeline proposal and why we needed to introduce this configuration option.
```

Comment:
Oh this link is going to the original PR for pipeline. I would move this to line 17? And replace this this the release notes for https://github.com/babel/babel/releases/tag/v7.0.0-beta.52?

Reply:
Oh, whoops, I copied your link from a comment without checking it. I intended to link to the PR that introduced the change. We could link to the release notes, altho we are saying "with the release of beta54", and then linking to "beta.52"?